src/patternfly/components/JumpLinks/examples/JumpLinks.md

```diff
@@ -342,6 +342,190 @@ beta: true
+    Jump to section
+  {{/jump-links-label}}
+  {{#> jump-links-list}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item jump-links-item--modifier="pf-m-current"}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Active section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+  {{/jump-links-list}}
+{{/jump-links}}
+```
+
+### Expanded
+```hbs
+{{#> jump-links jump-links--id="jump-links-expanded" jump-links--IsExpandable="true" jump-links--IsExpanded="true" jump-links--modifier="pf-m-vertical" jump-links--attribute='aria-label="Local"'}}
+  {{#> jump-links-label}}
+    Jump to section
+  {{/jump-links-label}}
+  {{#> jump-links-list}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item jump-links-item--modifier="pf-m-current"}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Active section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-link-text}}
+          Inactive section
+        {{/jump-links-link-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+  {{/jump-links-list}}
+{{/jump-links}}
+```
+
+### Expandable (responsive)
+```hbs
+{{#> jump-links jump-links--id="jump-links-expandable-responsive" jump-links--IsExpandable="true" jump-links--modifier="pf-m-vertical pf-m-expandable pf-m-non-expandable-on-md pf-m-expandable-on-lg pf-m-non-expandable-on-xl" jump-links--attribute='aria-label="Local"'}}
```

Comment:
Since `jump-links--IsExpandable="true"` here, do we still need `pf-m-expandable` in `jump-links--modifier`?

Reply:
Nice catch, updated!